include/seastar/http/url.hh

```diff
@@ -26,9 +26,9 @@ namespace seastar {
 namespace http {
 namespace internal {
 
-bool url_decode(const std::string_view& in, sstring& out);
+bool url_decode(const std::string_view in, sstring& out);
```

Comment:
nit: here and elsewhere - `std::string_view` is const by definition, so you shouldn't need to qualify it here.

Reply:
Makes sense, I removed `const` now